### PR TITLE
Feature/portal 22/show list collators

### PR DESCRIPTION
**Pull request by nicolasdanelon**

### CODE REVIEW

File: src/pages/collators/Collators.tsx [hunk 1]
```diff
@@ -16,10 +16,27 @@ enum filters {
...
+  owner: string[];
+  delegators: string[];
+  id: string;
+  status: string;
+  total: string;
 };
 
 export function Collators() {
   const [candidates, setCandidates] = useState<Candidate[]>([]);
+  const [candidatePool, setCandidatePool] = useState<CandidatePoolProps[]>([]);
```
Comment (gonzamontiel) [inline src/pages/collators/Collators.tsx:37]: why `CandidatePoolProps`, but not `CandidateProps`. I find it confusing. Maybe just remove `Props` from `CandidatePoolProps`?

File: src/pages/dashboard/Dashboard.tsx [hunk 2]
```diff
@@ -31,7 +31,7 @@ export function Dashboard() {
   const { free } = accountBalance;
 
   if (free) {
-    console.log("Free proc: ", toUnit(free) / 1000);
+    console.log("Free proc: ", toUnit(free));
```
Comment (gonzamontiel) [inline src/pages/dashboard/Dashboard.tsx:34]: is this log intended?

**Review by gonzamontiel — APPROVED** [review]
I left some comments but overall is fine.

**Comment by prayagd** [comment]
@nicolasdanelon Listing down few minor fixes
1. Second column is called `Total staked` not `Total bounded`
2. Last column is called `Delegated` not `Last block`
Otherwise all good.

**Pull request merged**